modules/lighthouse_service.js

```diff
@@ -229,6 +229,87 @@ const failDestinationPlateBeckman = async (form) => {
   }
 }
 
+/**
+ * Format the plate specs to the expected type
+ * @param {*} plateSpecs a list of objects e.g. [{numberOfPlates: 1, numberOfPositives: 2}, {numberOfPlates: 3, numberOfPositives: 4}]
+ * @returns {list} plateSpecsList a string represtation of the plate specs in a nested list e.g. [[1,2],[3,4]]
+ */
+const formatPlateSpecs = (plateSpecs) => {
+  return plateSpecs.map((plate) => { return [plate.numberOfPlates, plate.numberOfPositives] })
+}
+
+// Create a test run
+const generateTestRun = async (plateSpecs, addToDart) => {
+  const plateSpecsParam = formatPlateSpecs(plateSpecs)
+  try {
+    const url = `${config.privateRuntimeConfig.lighthouseBaseURL}/cherrypick-test-data`
+    const body = {
+      'plate_specs': plateSpecsParam,
+      'add_to_dart': addToDart,
+    }
+    const headers = { headers: { Authorization: config.privateRuntimeConfig.lighthouseApiKey } }
+
+    const response = await axios.post(url, body, headers)
+
+    return {
+      success: true,
+      runId: response.data._id,
+    }
+  } catch (error) {
+    return {
+      success: false,
+      error: error.response ? error.response.data._error.message + ": " + JSON.stringify(error.response.data._issues) : 'An unexpected error has occured'
+    }
+  }
+}
+
+// Get all test runs
+const getTestRuns = async (currentPage, perPage) => {
+  try {
+    // TODO: refactor path
```

Comment:
this

Reply:
You can do this with `URLSearchParams` if you want to do it:

```
const params = new URLSearchParams()
params.append("max_results", perPage)
params.append(...)

const url = "https://the-base-url/endpoint?" + params.toString()
```